src/typeMapper.js

```diff
@@ -1,7 +1,7 @@
 import { _ } from 'gridjs-react';
 
 export const typeRepresentationMapper = {
-  "img": (value) => _(<img src={value.value}></img>),
+  "img": (value) => _(<img alt="" src={value.value}></img>),
```

Comment:
This is not a solution. The `alt` has to actually contain something, else it's useless for the user.

Reply:
The ESLint suggests using an empty alt property for decorative images, I have now changed it in [commit a0ee7da](https://github.com/SolidLabResearch/generic-data-viewer/pull/40/commits/a0ee7dabe49daae2ee149042de01a387f4508034) which indeed seems much better.